src/pydata_sphinx_theme/assets/styles/extensions/_pydata.scss

```diff
@@ -6,3 +6,9 @@
 .xr-wrap[hidden] {
   display: block !important;
 }
+
+// ipywidgets
+.jp-OutputArea-output.lm-Widget {
+  // override overflow:hidden rule from Lumino (.lm-Widget) to allow scrolling
+  overflow: auto;
```

Comment:
I'm not entirely sure if I should set `auto` for both x and y or just x, but I decided to follow JupyterLab's lead, though to be honest I'm not entirely sure if the scenario described in https://github.com/jupyterlab/jupyterlab/pull/6104 is applicable or not to sites built with PyData Sphinx Theme.